src/components/Layout/AsideMenu/MenuItems.jsx

```diff
@@ -0,0 +1,78 @@
+import { MenuList, Icon, MenuItem, Box, Text } from '@chakra-ui/react';
+import React from 'react';
+import { AiOutlineTeam } from 'react-icons/ai';
+import { MdOutlinePending } from 'react-icons/md';
+import { RiQuestionnaireLine, RiInformationLine, RiProfileLine } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+
+function MenuWrap({ menuType, children }) {
+  if (menuType === 'mobile') {
+    return <MenuList>{children}</MenuList>;
+  } else {
+    return <Box>{children}</Box>;
+  }
+}
+
+function MenuLink({ menuType, children }) {
+  if (menuType === 'mobile') {
+    return <MenuItem>{children}</MenuItem>;
+  } else {
+    return <Box mt=".7rem">{children}</Box>;
+  }
+}
+
+function MenuItems({ menuType }) {
+  let nav;
+
+  switch (menuType) {
+    default:
+      nav = (
+        <MenuWrap menuType={menuType}>
+          <Link to="/pending-approval">
```

Comment:
Good practice is to create an array with data to render the menu, and render it like that, so we don't write jsx for each menu item separately:

```js
const menuItems = [{
    href: '/pending-approval',
    icon: MdOutlinePending,
    text: 'Pending for approval',
}, ...];
```

and render it

```js
{menuItems.map(({text, icon, href}) => (
    <Link to={href} .....
))
```